src/content/docs/cypher/expressions/node-rel-functions.md

```diff
@@ -1,15 +1,15 @@
 ---
-title: Node/Rel Functions
-description: Node/Rel functions are used to get information about nodes and relationships.
+title: Node & relationship Functions
```

Comment:
```suggestion
title: Node & Relationship Functions
```

Reply:
We're following sentence case, not title case for headers (this is common in blogs and docs). In academic publications is where you normally capitalize all headers, it doesn't read well on websites.